gwtmockito/src/main/java/com/google/gwtmockito/GwtMockitoTestRunner.java

```diff
@@ -78,6 +78,8 @@
 import java.util.LinkedList;
 import java.util.List;
 
+import static com.google.gwtmockito.impl.ThreadLocalCleaner.cleanUpThreadLocalValues;
```

Comment:
Probably better to reference this directly instead of static-importing it since it's just used in the one place.